packages/components/typography/src/typography.css

```diff
@@ -1,5 +1,9 @@
 /* General........................................................ */
 
+:host {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)

Reply:
@ehats 
Can you disable this one for this repo?